src/main/java/com/minecolonies/coremod/colony/buildings/BuildingDeliveryman.java

```diff
@@ -0,0 +1,116 @@
+package com.minecolonies.coremod.colony.buildings;
+
+import com.minecolonies.blockout.views.Window;
+import com.minecolonies.coremod.client.gui.WindowHutDeliveryman;
+import com.minecolonies.coremod.colony.CitizenData;
+import com.minecolonies.coremod.colony.Colony;
+import com.minecolonies.coremod.colony.ColonyView;
+import com.minecolonies.coremod.colony.jobs.AbstractJob;
+import com.minecolonies.coremod.colony.jobs.JobDeliveryman;
+import io.netty.buffer.ByteBuf;
+import net.minecraft.nbt.NBTTagCompound;
+import net.minecraft.util.math.BlockPos;
+import org.jetbrains.annotations.NotNull;
+
+/**
+ * Class of the warehouse building.
+ */
+public class BuildingDeliveryman extends AbstractBuildingWorker
+{
+
+    private static final String DELIVERYMAN = "Deliveryman";
+
+    private static final String TAG_DELIVERY = "delivery";
+
+    /**
+     * Instantiates a new warehouse building.
+     *
+     * @param c the colony.
+     * @param l the location
+     */
+    public BuildingDeliveryman(final Colony c, final BlockPos l)
+    {
+        super(c, l);
+    }
+
+    @NotNull
+    @Override
+    public String getSchematicName()
+    {
+        return DELIVERYMAN;
+    }
+
+    @Override
+    public int getMaxBuildingLevel()
+    {
+        return 4;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Assign this magic number 4 to a well-named constant, and use the constant instead. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS109)